notebooks/how_to_handle_periods.ipynb

```diff
@@ -722,13 +554,16 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "[ 519.96002197]\n"
+      "[519.18]\n"
      ]
     }
    ],
    "source": [
-    "af_march_to_july = simulation.calculate_add('af' ,periods.period('month', '2015-04', 4))\n",
-    "print af_march_to_july"
+    "af_april_to_july = simulation.calculate_add(\n",
+    "    'af', \n",
+    "    period = periods.Period(('month', periods.Instant((2014, 3, 1)), 4))\n",
```

Comment:
April for `(2014, 3, 1)`? July for `(2014, 3, 1) + 4`? 🙃

Reply:
4 is the length of the period in terms of unit